src/templates/bet/analysis.html.tt

```diff
@@ -0,0 +1,65 @@
+<div id="trading_bottom_content">
+  <div class="content-tab-container page-section">
+    <div class="tab-menu">
+      <div class="tab-menu-wrap grd-container">
+        <ul id="betsBottomPage" class="tm-ul">
+          <li id="tab_portfolio" class="tm-li invisible first">
+            <a href="#tab_portfolio" class="tm-a first"></a>
+          </li>
+          <li id="tab_graph" class="tm-li first">
+            <a href="#tab_graph" class="tm-a first"></a>
+          </li>
+          <li id="tab_explanation" class="tm-li active">
+            <a href="" class="tm-a"></a>
+          </li>
+          <li id="tab_last_digit" class="invisible tm-li">
+            <a href="#" class="tm-a"></a>
+          </li>
+          <li id="tab_japan_info" class="invisible tm-li last">
+            <a href="#" class="tm-a"></a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div class="tab-content grd-container">
+      <div class="tab-content-wrapper" id="bet_bottom_content">
+        <div id="tab_portfolio-content" class="toggle-content invisible "></div>
+        <div id="tab_graph-content" class="toggle-content invisible">
+          <p class="error-msg" id="chart-error"></p>
+          <div id="trade_live_chart">
+            <table width="600px" align="center">
+              <tr id="highchart_duration">
```

Comment:
please remove the highchart_duration tr element, as it was removed with the last merge.